content/projects/index.adoc

```diff
@@ -121,6 +121,19 @@ Jenkins X brings along a selection of excellent pre-integrated open source tools
 To use Jenkins X you don't need to install Jenkins separately.
 
 link:jenkins-x[*Learn more*]
+  
+---
```

Comment:
Without medata it likely causes the build to fail

Reply:
not sure what you mean by that... I used the same format Jenkins X uses, just added one more entry